docs/source/json-api/index.rst

```diff
@@ -266,12 +266,16 @@ The JSON API essentially performs two separate tasks:
 
 .. note:: By default, the Daml Sandbox does not does not require access tokens. However, you still need to provide a party-specific access token when submitting commands or queries as a party. The token will not be validated in this case but it will be decoded to extract information like the party submitting the command.
 
-Party-specific Access Tokens
-----------------------------
+Party-specific Requests
+-----------------------
 
-Party-specific requests, i.e., command submissions and queries, require a JWT with some additional restrictions compared to general :ref:`Daml ledger access token formats <access-token-formats>`. For command submissions, ``actAs`` must contain at least one party and ``readAs`` can contain 0 or more parties. Queries require at least one party in either ``actAs`` or ``readAs``. The application id and ledger id are mandatory. HTTP requests pass the token in a header, while WebSocket requests pass the token in a subprotocol.
+Party-specific requests, i.e., command submissions and queries, require a JWT that demonstrates the bearer's authorization to issue the request.
 
-.. note:: While the JSON API receives the token it doesn't validate it itself. Upon receiving a token it will pass it, and all data contained within the request, on to the Ledger API's AuthService which will then determine if the token is valid and authorized. However, the JSON API does decode the token to extract the ledger id, application id and party so it requires that you use the JWT format documented below.
+If the underlying ledger supports :ref:`user management <user-service>` (this includes Canton and the sandbox), the only restriction is imposed by the authorization: the bearer's user must have the rights to perform the operation. The HTTP JSON API will parse the token, read the user id and fill it into the Ledger API request's application id field.
```

Comment:
not certain about the part
> read the user id and fill it into the Ledger API request's application id field.

We do read the user id from the token but we only do so in endpoints that require the user id for calls which so far are only a few user management endpoints (i.e. /user to get user information about the user associated with the token)

Reply:
See https://github.com/digital-asset/daml/blob/728f03c85546ba88111114a8b3b17fc91aaa9adb/ledger-service/http-json/src/main/scala/com/digitalasset/http/EndpointsCompanion.scala#L159

You also use that on command submissions to fill in the application id as far as I know.

Reply:
@realvictorprm Is @cocreature's point valid or not? You wrote the code, I assume you're in the best position to tell whether I'm writing something correct or not.

Reply:
@cocreature definitely is right (as the code shows) and with me additionally remembering that this is the case because we need to provide an application id such that we decided on to use the user-id for that (all makes sense again to me xD). Very sorry for the confusion.

Reply:
No problem, thanks for the clarification.